x-pack/legacy/plugins/actions/server/actions_config.mock.ts

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
hahaha I just created one of these for another PR!!!!

Reply:
It was the obvious refactor :)